addons/stock/wizard/make_procurement_product.py

```diff
@@ -36,6 +38,30 @@ def onchange_product_id(self, cr, uid, ids, prod_id):
         'qty': lambda *args: 1.0,
     }
 
+    def fields_view_get(self, cr, uid, view_id=None, view_type='form', context=None, toolbar=False, submenu=False):
```

Comment:
@tde-banana-odoo ,
As per spec
When click on ***Procurement Request*** button, Then 
1: If view is product.template, then should show only variants of current product template's on wizard.
-> If product template has one variant, then product should invisible on wizard,
See here: https://drive.google.com/file/d/0BxqHe5rtIxY0YUl3SEVObmMwXzQ/view

2: If view is product.product then product field should read-only on wizard,